### PR TITLE
Remove nft trending and partners strings

### PR DESCRIPTION
**Pull request by vetkamak**

### CODE REVIEW

File: src/en.json [hunk 1]
```diff
@@ -37,20 +37,7 @@
...
-                "grandbazar": {
-                    "description": "Rare digital items",
-                    "title": "Grandbazar"
-                },
-                "thirdPlace": {
-                    "description": "Real-world art objects",
-                    "title": "Third Place"
-                }
-            },
+            "empty": "No collections yet",
```
Comment (vetkamak) [inline src/en.json]: @AnnStepanova could you please provide more appropriate text for the empty Collectibles screen?

File: src/ru.json [hunk 2]
```diff
@@ -37,20 +37,7 @@
...
-                "grandbazar": {
-                    "description": "Rare digital items",
-                    "title": "Grandbazar"
-                },
-                "thirdPlace": {
-                    "description": "Real-world art objects",
-                    "title": "Third Place"
-                }
-            },
+            "empty": "No collections yet",
```
Comment (vetkamak) [inline src/ru.json]: @AnnStepanova and translation for it as well :)

File: src/en.json (hunk 1 above)
Comment (AnnStepanova) [inline src/en.json]:
```suggestion
            "empty": "Here you can view and manage your NFTs after you start collecting them",
```

File: src/ru.json (hunk 2 above)
Comment (AnnStepanova) [inline src/ru.json]:
```suggestion
            "empty": "Здесь вы сможете любоваться и управлять своей коллекцией NFT, если решите ее собирать",
```

File: src/ko.json [hunk 3]
```diff
@@ -37,20 +37,7 @@
...
-                "grandbazar": {
-                    "description": "희귀 디지털 아이템",
-                    "title": "Grandbazar"
-                },
-                "thirdPlace": {
-                    "description": "현실의 예술품",
-                    "title": "Third Place"
-                }
-            },
+            "empty": "Here you can view and manage your NFTs after you start collecting them",
```
Comment (vetkamak) [inline src/ko.json]: @kkkhesha could you translate it, please? :)

Comment (kkkhesha) [inline src/ko.json]:
```suggestion
            "empty": "NFT 수집을 시작한 후 여기에서 NFT를 보고 관리할 수 있습니다.",
```

**Review by sertony — APPROVED** [review]

**Review by AnnStepanova — APPROVED** [review]

**Pull request merged**